Python/Product/PythonTools/PythonTools/Environments/VirtualEnv.cs

```diff
@@ -405,8 +405,13 @@ IInterpreterRegistryService service
             interpExe = Path.GetFileName(baseInterpreter.Configuration.InterpreterPath);
             winterpExe = Path.GetFileName(baseInterpreter.Configuration.GetWindowsInterpreterPath());
             var scripts = new[] { "Scripts", "bin" };
+
+            // new versions of python >= 3.8 from the windows store have a different executable name (ie. python3.8.exe) in the 
+            // registry InstallPath, compared to what is in the venv folder (python.exe) so search for both if interpExe is not found.
             interpExe = PathUtils.FindFile(prefixPath, interpExe, firstCheck: scripts);
+            interpExe = interpExe == null ? PathUtils.FindFile(prefixPath, "python.exe", firstCheck: scripts) : interpExe;
```

Comment:
you could do it in one statement instead of 2

interpExe = PathUtils.FindFile(...) ?? PathUtils.FindFile(...);